src/bomf/model/__init__.py

```diff
@@ -2,60 +2,11 @@
 general data models for migrations
 """
 
-import enum
 import uuid
 from abc import ABC
-from typing import TypeVar, Union
 
-from bo4e.bo.geschaeftsobjekt import Geschaeftsobjekt
-from bo4e.com.com import COM
 from pydantic import BaseModel, Field  # pylint: disable=no-name-in-module
 
-_SpecificBusinessObject = TypeVar("_SpecificBusinessObject", bound=Geschaeftsobjekt)
-"""
-an arbitrary but fixed business object type
-"""
-
-_SpecificCom = TypeVar("_SpecificCom", bound=COM)
-"""
-an arbitrary but fixed COM type
-"""
-
-Bo4eTyp = Union[_SpecificBusinessObject, _SpecificCom]  # pylint: disable=invalid-name
-
-
-# pylint:disable=too-few-public-methods
-class BusinessObjectRelation(BaseModel):
```

Comment:
technically, removing this class is a breaking change.
but we don't use it anywhere outside this repo:
https://github.com/search?q=org%3AHochfrequenz+BusinessObjectRelation&type=code